### ISSUE TITLE
state_dict problem

### ISSUE THREAD

**Issue by tranquoclongt1** [amadeuzou/vrn-pytorch#1]
I've met this problem but not find any solution. It seems that there's no label 'state_dict'? Could you help me out? Thank you :+1: 
![image](https://user-images.githubusercontent.com/16191939/37329920-033be94a-26d2-11e8-99de-17307bfae697.png)

**Comment by jandremarais** [comment]
I swapped the face_alignment directory with the latest one at https://github.com/1adrianb/face-alignment which solved this problem for me.
